dna/clutter/clutter.js

```diff
@@ -241,17 +192,25 @@ function doGetLink(base,tag) {
     return links_filled;
 }
 
-// ==============================================================================
-// CALLBACKS: Called by back-end system, instead of front-end app or UI
-// ===============================================================================
+function anchor(anchorType, anchorText){
+  return call('anchors', 'anchor', {anchorType: anchorType, anchorText: anchorText}).replace(/"/g, '')
+}
+
+function anchorHash(appKeyHash){
```

Comment:
why bother taking appKeyHash as an input if you can just retrieve it within the function?

Reply:
oh, I get it. I would say just rename it `hash` so that it's more generic and not misLeading